src/Symfony/Component/Validator/Constraints/BackedEnumValue.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\Exception\ConstraintDefinitionException;
+
+/**
+ * Validates that a value is one of an enum.
+ *
+ * @author Aurélien Pillevesse <aurelienpillevesse@hotmail.fr>
+ */
+#[\Attribute(\Attribute::TARGET_PROPERTY | \Attribute::TARGET_METHOD | \Attribute::IS_REPEATABLE)]
+class BackedEnumValue extends Constraint
+{
+    public const NO_SUCH_CHOICE_ERROR = '53dcc1b1-a8dd-4813-baa5-b8486ff56447';
+
+    protected const ERROR_NAMES = [
+        self::NO_SUCH_CHOICE_ERROR => 'NO_SUCH_CHOICE_ERROR',
+    ];
+
+    /**
+     * @param class-string<\BackedEnum> $type   the type of the enum
+     * @param array                     $only   the cases that should be considered valid
+     * @param array                     $except the cases that should be considered invalid
```

Comment:
Those should describe the type of values in the array (are they instance of the enums or their values ?)